model/generate.xml

```diff
@@ -479,6 +480,7 @@
 
   <command symbol="script-decode" formerly="showscript" output="string" multipleX="true" category="TRANSACTION" description="Decode a script to plain text tokens.">
     <argument name="BASE16" fileX="true" stdin="true" type="base16" description="The Base16 script. If not specified the script is read from STDIN."/>
+    <option name="flags" type="uint32_t" configuration="wallet.rule_fork_flags" description="The rule_fork flags for decoding, defaults to all" />
```

Comment:
"The rule fork flags, defaults to all (4294967295)."

"decoding" is inherent, remove rule_fork underscore, add period and numeric default.